rust/private/rustc.bzl

```diff
@@ -857,3 +864,11 @@ error_format = rule(
     implementation = _error_format_impl,
     build_setting = config.string(flag = True),
 )
+
+def _extra_codegen_impl(ctx):
+    return ExtraCodegenInfo(extra_codegen = ctx.build_setting_value)
+
+extra_codegen = rule(
```

Comment:
Nit: can you add a doc and plug this into `./docs`

Reply:
Updated the docs for this (and `error_format`, while I'm at it)

Reply:
It still seems like there's a missing `docs` attribute and it doesn't seem like this is appearing in docs. It'd be nice to have this exposed there so people can easily discover it